recipes/las-geoh5/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set name = "las-geoh5" %}
+{% set version = "0.2.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/MiraGeoscience/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: 1c20e6bb021516ef89de8551dc1f47f944bc98ea83d3da393d13e71469add43d
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m conda config --add channels kinverarity && {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
```

Comment:
Hi @conda-forge/help-python, could you help me ? I really need to add another conda channel `kinverarity`. Have you any idea, how to do it?

Reply:
We do not support mixing with third party channel b/c we don't know and cannot verify what is in there. You must build everything you need in conda-forge.

Reply:
Thank you @ocefpaf . I'm gonna publish `lasio` package on conda-forge. Thank you